diff --git a/platform_core/platform/emdrv/dmadrv/src/dmadrv.c b/platform_core/platform/emdrv/dmadrv/src/dmadrv.c
index dd45751..06234c6 100755
--- a/platform_core/platform/emdrv/dmadrv/src/dmadrv.c
+++ b/platform_core/platform/emdrv/dmadrv/src/dmadrv.c
@@ -320,6 +320,12 @@ Ecode_t DMADRV_Init(void)
     chTable[i].allocated = false;
   }
 
+  for (int i = 0; i < EMDRV_DMADRV_DMA_CH_COUNT; i++ ) {
+    if ( (1U << i) & (uint32_t)(ZWAVE_LDMA_RESERVED_CHANNELS) ) {
+      chTable[i].allocated = true;
+    }
+  }
+
 #if defined(EMDRV_DMADRV_UDMA)
   NVIC_SetPriority(DMA_IRQn, EMDRV_DMADRV_DMA_IRQ_PRIORITY);
   dmaInit.hprot        = 0;
@@ -1036,7 +1042,7 @@ Ecode_t DMADRV_TransferRemainingCount(unsigned int channelId,
  * @brief
  *  An interrupt handler for LDMA.
  ******************************************************************************/
-void LDMA_IRQHandler(void)
+void dmadrv_LDMA_IRQHandler(void)
 {
   bool stop;
   ChTable_t *ch;
@@ -1047,6 +1053,7 @@ void LDMA_IRQHandler(void)
   /* Get all pending and enabled interrupts. */
   pending  = LDMA->IF;
   pending &= LDMA->IEN;
+  pending &= ~(uint32_t)(ZWAVE_LDMA_RESERVED_CHANNELS);
 
   /* Check for LDMA error. */
   if ( pending & LDMA_IF_ERROR ) {
