diff --git a/platform_core/platform/service/dma_manager/src/sl_dma_manager_hal_ldma.c b/platform_core/platform/service/dma_manager/src/sl_dma_manager_hal_ldma.c
--- a/platform_core/platform/service/dma_manager/src/sl_dma_manager_hal_ldma.c
+++ b/platform_core/platform/service/dma_manager/src/sl_dma_manager_hal_ldma.c
@@ -275,7 +275,7 @@ void sli_dma_manager_hal_set_pending_errors(uint8_t channel_nbr, uint32_t errors
 /***************************************************************************//**
  * Interrupt handler for LDMA module.
  ******************************************************************************/
-void LDMA_IRQHandler(void)
+void dmadrv_LDMA_IRQHandler(void)
 {
   uint32_t pending;
   uint32_t pending_done;
