diff --git a/config/sl_cpc_drv_secondary_uart_usart_vcom_config.h b/config/sl_cpc_drv_secondary_uart_usart_vcom_config.h
index 9f05293..20ea70d 100644
--- a/config/sl_cpc_drv_secondary_uart_usart_vcom_config.h
+++ b/config/sl_cpc_drv_secondary_uart_usart_vcom_config.h
@@ -61,20 +61,15 @@
 #define SL_CPC_DRV_UART_VCOM_PERIPHERAL          USART0
 #define SL_CPC_DRV_UART_VCOM_PERIPHERAL_NO       0
 
-// USART0 TX on PD13
-#define SL_CPC_DRV_UART_VCOM_TX_PORT             gpioPortD
-#define SL_CPC_DRV_UART_VCOM_TX_PIN              13
-#define SL_CPC_DRV_UART_VCOM_TX_LOC              21
+// USART0 TX on PA1
+#define SL_CPC_DRV_UART_VCOM_TX_PORT             gpioPortA
+#define SL_CPC_DRV_UART_VCOM_TX_PIN              1
+#define SL_CPC_DRV_UART_VCOM_TX_LOC              1
 
-// USART0 RX on PD14
-#define SL_CPC_DRV_UART_VCOM_RX_PORT             gpioPortD
-#define SL_CPC_DRV_UART_VCOM_RX_PIN              14
-#define SL_CPC_DRV_UART_VCOM_RX_LOC              21
-
-// USART0 CTS on PD15
-#define SL_CPC_DRV_UART_VCOM_CTS_PORT            gpioPortD
-#define SL_CPC_DRV_UART_VCOM_CTS_PIN             15
-#define SL_CPC_DRV_UART_VCOM_CTS_LOC             19
+// USART0 RX on PA0
+#define SL_CPC_DRV_UART_VCOM_RX_PORT             gpioPortA
+#define SL_CPC_DRV_UART_VCOM_RX_PIN              0
+#define SL_CPC_DRV_UART_VCOM_RX_LOC              31
 
 // [USART_SL_CPC_DRV_UART_VCOM]$
 // <<< sl:end pin_tool >>>
