diff --git a/config/sl_cpc_drv_secondary_uart_usart_vcom_config.h b/config/sl_cpc_drv_secondary_uart_usart_vcom_config.h
index 9a63e28..b56bfcb 100644
--- a/config/sl_cpc_drv_secondary_uart_usart_vcom_config.h
+++ b/config/sl_cpc_drv_secondary_uart_usart_vcom_config.h
@@ -59,28 +59,18 @@
 // <<< sl:start pin_tool >>>
 // <usart signal=TX,RX,(CTS),(RTS)> SL_CPC_DRV_UART_VCOM
 // $[USART_SL_CPC_DRV_UART_VCOM]
-#define SL_CPC_DRV_UART_VCOM_PERIPHERAL                 USART0
-#define SL_CPC_DRV_UART_VCOM_PERIPHERAL_NO              0
+#define SL_CPC_DRV_UART_VCOM_PERIPHERAL          USART0
+#define SL_CPC_DRV_UART_VCOM_PERIPHERAL_NO       0
 
-// USART TX
-#define SL_CPC_DRV_UART_VCOM_TX_PORT                    gpioPortD
-#define SL_CPC_DRV_UART_VCOM_TX_PIN                     13
-#define SL_CPC_DRV_UART_VCOM_TX_LOC                     22
+// USART0 TX on PA1
+#define SL_CPC_DRV_UART_VCOM_TX_PORT             gpioPortA
+#define SL_CPC_DRV_UART_VCOM_TX_PIN              1
+#define SL_CPC_DRV_UART_VCOM_TX_LOC              1
 
-// USART RX
-#define SL_CPC_DRV_UART_VCOM_RX_PORT                    gpioPortD
-#define SL_CPC_DRV_UART_VCOM_RX_PIN                     14
-#define SL_CPC_DRV_UART_VCOM_RX_LOC                     20
-
-// USART CTS
-#define SL_CPC_DRV_UART_VCOM_CTS_PORT                    gpioPortD
-#define SL_CPC_DRV_UART_VCOM_CTS_PIN                     15
-#define SL_CPC_DRV_UART_VCOM_CTS_LOC                     20
-
-// USART RTS
-#define SL_CPC_DRV_UART_VCOM_RTS_PORT                    gpioPortD
-#define SL_CPC_DRV_UART_VCOM_RTS_PIN                     16
-#define SL_CPC_DRV_UART_VCOM_RTS_LOC                     20
+// USART0 RX on PA0
+#define SL_CPC_DRV_UART_VCOM_RX_PORT             gpioPortA
+#define SL_CPC_DRV_UART_VCOM_RX_PIN              0
+#define SL_CPC_DRV_UART_VCOM_RX_LOC              31
 
 // [USART_SL_CPC_DRV_UART_VCOM]$
 // <<< sl:end pin_tool >>>
