diff --git a/util/third_party/openthread/src/lib/spinel/radio_spinel.cpp b/util/third_party/openthread/src/lib/spinel/radio_spinel.cpp
index d89958c20..e5f7636aa 100644
--- a/util/third_party/openthread/src/lib/spinel/radio_spinel.cpp
+++ b/util/third_party/openthread/src/lib/spinel/radio_spinel.cpp
@@ -2170,12 +2170,16 @@ void RadioSpinel::RestoreProperties(void)
             Set(SPINEL_PROP_RCP_MAC_FRAME_COUNTER, SPINEL_DATATYPE_UINT32_S, mMacFrameCounter + kFrameCounterGuard));
     }
 
+    ClearSrcMatchShortEntries();
+
     for (int i = 0; i < mSrcMatchShortEntryCount; ++i)
     {
         SuccessOrDie(
             Insert(SPINEL_PROP_MAC_SRC_MATCH_SHORT_ADDRESSES, SPINEL_DATATYPE_UINT16_S, mSrcMatchShortEntries[i]));
     }
 
+    ClearSrcMatchExtEntries();
+
     for (int i = 0; i < mSrcMatchExtEntryCount; ++i)
     {
         SuccessOrDie(
