diff --git a/zwave/ZAF/CommandClasses/ColorSwitch/src/CC_ColorSwitch.c b/zwave/ZAF/CommandClasses/ColorSwitch/src/CC_ColorSwitch.c
index 895e9a4..9208b0d 100644
--- a/zwave/ZAF/CommandClasses/ColorSwitch/src/CC_ColorSwitch.c
+++ b/zwave/ZAF/CommandClasses/ColorSwitch/src/CC_ColorSwitch.c
@@ -37,6 +37,13 @@ ZW_WEAK void cc_color_switch_refresh_cb(void)
 {
 }
 
+extern uint8_t cc_color_switch_get_default_value(s_colorComponent* colorComponent);
+ZW_WEAK uint8_t cc_color_switch_get_default_value(s_colorComponent* colorComponent)
+{
+  (void) colorComponent;
+  return 0xff;
+}
+
 typedef void (*color_callback_t)(s_colorComponent *colorComponent);
 
 /**
@@ -58,7 +65,16 @@ static void init_and_reset(void)
   }
   for (uint8_t i = 0; i < colorsSupportedCount; i++) {
     ZAF_Actuator_Init(&pColorComponents[i].obj, 0, 0xFF, 20, durationDefault, &CC_ColorSwitch_ColorChanged_cb);
-    ZAF_Actuator_Set(&pColorComponents[i].obj, 0xFF, 0);
+    uint8_t color_component_id = findColorComponentIndexByObj(&pColorComponents[i].obj);
+    if (!cc_color_switch_read(color_component_id, &pColorComponents[i]))
+    {
+      // If the color was not found in NVM, set it to default value
+      ZAF_Actuator_Set(
+        &pColorComponents[i].obj,
+        cc_color_switch_get_default_value(&pColorComponents[i]),
+        0
+      );
+    }
     cc_color_switch_write((uint8_t) i, &pColorComponents[i]);
   }
 }
